base/applications/regedit/main.c

```diff
@@ -2,6 +2,7 @@
  * Regedit main function
  *
  * Copyright (C) 2002 Robert Dickenson <robd@reactos.org>
+ * Copyright (C) 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```

Comment:
Not sure adding copyrights are relevant when a one-line addition is done in a large file...

Reply:
Deleted.